### DISCUSSION TITLE
Difference between the Micro and Distroless containers

### DISCUSSION THREAD

**Discussion by kjq** [quarkusio/quarkus#24376] [Q&A] [answered]
What are the differences between using the micro vs. distroless container?  Are there any advantages of using one over the other?

**Comment by cescoffier** [comment, marked as answer]
Quarkus Distroless is a base image relying on the Google distroless effort. It's Debian based and provides only the bare minimum.

Quarkus micro runtime is a base image using Ubi micro (so Redhat / Centos). It's a bit bigger than distroless but eases the maintenance (you can easily add more content), and has been curated for containers.
